src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        System.out.println("Hello there! I'm Duke.\nWhat can I do for you?\n");
+
+        Task[] taskList = new Task[100];
```

Comment:
Perhaps you can have a task manager class to handle the tasks instead of having everything inside main?